scripts/typescript_checks.py

```diff
@@ -432,11 +432,13 @@
     'core/templates/pages/exploration-player-page/services/state-classifier-mapping.service.spec.ts',
     'core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.spec.ts',
     'core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.ts',
+    'core/templates/pages/facilitator-dashboard-page/facilitator-dashboard-page.import.ts',
     'core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.component.spec.ts',
     'core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.component.ts',
     'core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.spec.ts',
     'core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.ts',
     'core/templates/pages/learner-dashboard-page/learner-dashboard-page.import.ts',
+    'core/templates/pages/learner-group-pages/create-group/create-learner-group-page.import.ts',
```

Comment:
@vojtechjelinek is adding these files to the string typing exclusion list okay?

Reply:
Hm, I don't think it's ok, actually. @paprajapati9 why do these need to be added? Typing them should be straightforward, I think (/cc @JeeveshGarg for info)

Reply:
Hmm, actually I did not know how to type it and I did not see any example for typings for .import.ts files so I had added them here with the intention of getting some help from reviewers about how to type this file. Will contact Jeevesh and will try to type them. Thanks!

Reply:
Yes @paprajapati9 feel free to add import file in exclude list. As import file have dependencies, which we need to solve first.

Reply:
OK @JeeveshGarg. Thanks for confirming!

Reply:
Still need approval by @vojtechjelinek since they're in charge of the typing effort

Reply:
LGTM!